src/styles/blocks/body.scss

```diff
@@ -0,0 +1,7 @@
+@use "../utils/variables" as *;
+
+.body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: $font-weight-normal;
```

Comment:
The variable `$font-weight-normal` is used here, but it is not defined or imported from the `variables` file. Please ensure that this variable is defined in the `../utils/variables` file or replace it with a valid font-weight value like `400`.